te2rules/adapter.py

```diff
@@ -229,3 +233,121 @@ def _convert(self) -> DecisionTree:
 
         root_node = nodes[0]
         return root_node
+
+
+class XgboostXGBClassifierAdapter:
+    """
+    Class to convert xgboost.sklearn.XGBClassifier
+    into a te2rules.tree.RandomForest object.
+
+    Usage:
+    adapter = XgboostXGBClassifierAdapter(model, feature_names)
+    adapted_model = adapter.random_forest
+    """
+
+    def __init__(self, xgb_model: XGBClassifier, feature_names: List[str]):
+        self.xgb_model = xgb_model
+        self.feature_names = feature_names
+
+        # TODO: n0 and n1 are not accessible from the xgboost model itself, in this way bias term will be 0
```

Comment:
It is true that xgboost library doesn't use bias and weights explicitly. `self.bias` can be directly set to 0 without n0, n1. These variables don't get used anywhere.